tic-tac-toe.js

```diff
@@ -1,7 +1,221 @@
 function TicTacToe() {
+  this.board = [["","",""], ["","",""], ["","",""]];
+  this.count = 0;
+  }
 
-}
+  var checkSpot = function(col, row, sign , board) {
+    if (board[col][row] === sign) {
+      return true;
+    } else {return false;}
+  }
 
-TicTacToe.prototype = {
-  
-}
+TicTacToe.prototype.CheckRow = function(col, row, sign) {
+    if (row === 0) {
```

Comment:
The indentation of this conditional vs the `{}` that are associated with each condition do not seem to match up. Watch out for creating conditionals without the `{}` because this can cause unexpected behavior.